packages/kotti-ui/source/kotti-field-select/components/GenericSelectField.vue

```diff
@@ -14,12 +14,14 @@
 					<KtTag
 						v-for="option in visibleSelectedTags"
 						:key="option.value"
+						class="kt-field-select__input-and-tags__tag"
```

Comment:
doesn't follow BEM convention, will edit after merging the stylelint MR
team will discuss this, this week in the chapter meet